README.md

```diff
@@ -83,7 +83,7 @@ docs = loader.lazy_load()
 ```
 
 See the
-full [Document Loader](https://github.com/googleapis/langchain-google-el-carro-python/tree/main/docs/doc_loader.md)
+full [Document Loader](https://github.com/googleapis/langchain-google-el-carro-python/tree/main/docs/doc_loader.ipynb)
```

Comment:
Can you update the file name to document_loader.ipynb so we are consistent across repos? Thanks!

Reply:
Done